public/js/employee/module.js

```diff
@@ -149,7 +150,13 @@ angular.module('ShoutOut.Employee.Controllers', [])
         if ($scope.survey.questions[i].type === 'slider') {
           $scope.survey.questions[i].response = $("#question_" + i).val();
         } else if ($scope.survey.questions[i].type === 'select') {
-          $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
+          if ( $scope.survey.questions[i].selectedValue != undefined) {
+            $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
+          }
+          else {
+            $scope.survey.questions[i].response = "Not Applicable";
+          }
```

Comment:
Just a heads up I believe this is incorrect, if its undefined (line 153), its going to assign the response to selectedValue, which is '1', '2', or '3'. So the else statement on line 157 should be assigning '3' I believe.